### PR TITLE
[Merged by Bors] - feat: substs tactic

### PR DESCRIPTION
**Pull request by evanlohn**
Added the substs tactic, which just applies subst to multiple hypotheses at once.

This is my first PR to mathlib; let me know what needs to change!

### CODE REVIEW

**Comment by digama0** [comment]
you need to add `import Mathlib.Tactic.Substs` to `Mathlib.lean`

File: Mathlib/Tactic/Substs.lean [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+namespace Mathlib.Tactic.Substs
+open Lean Meta Elab
+open Tactic
+
+syntax (name := substs) "substs" (colGt ppSpace ident)* : tactic
+
+macro_rules
+| `(tactic| substs $xs:ident*) =>
+  `(tactic| ($[subst $xs;]*))
```
Comment (digama0) [inline Mathlib/Tactic/Substs.lean:11]:
```suggestion
| `(tactic| substs $xs:ident*) => `(tactic| ($[subst $xs;]*))
```

File: Mathlib/Tactic/Substs.lean [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import Lean
```
Comment (digama0) [inline Mathlib/Tactic/Substs.lean:1]: needs file header

File: Mathlib/Tactic/Substs.lean [hunk 3]
```diff
@@ -11,5 +11,8 @@ open Tactic
 
 syntax (name := substs) "substs" (colGt ppSpace ident)* : tactic
 
+/--
+Applies the `subst` tactic to all given hypotheses from left to right
+-/
```
Comment (digama0) [inline Mathlib/Tactic/Substs.lean:16]: The comment goes on the `syntax` (that's the doc string that normally shows up when hovering in user code). Also `.` at the end of the sentence.

**Comment by gebner** [comment]
Thanks!

Please remove the "awaiting-author" label in the future when you've applied the changes from code review.

**Pull request closed without merging**